lib/bankaccounts/account.rb

```diff
@@ -0,0 +1,53 @@
+module BankAccounts
+
+  class Account
+    attr_accessor :id, :min_balance, :initial_balance, :balance, :fee
```

Comment:
Great use of inheritance to define accessors and member variables. :100: 

I do have a question though: Why have accessors defined for `min_balance`, `initial_balance`, and `fee`? Will the users of the classes need access to those attributes?